Source/core/TokenizedStringList.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+#include "Module.h"
+
+namespace WPEFramework {
+namespace Core {
+
+    template <char SEPARATOR, bool REMOVE_WHITESPACES>
```

Comment:
I know we do not actively test/use the widechar (wchar_t) but if for some reason we should, all code is prepared for it, by using preprocessor definitions, and it might work (or at least we have a lot less work :-) ). 
To be inline with that possibility, pleas make the char -> TCHAR which for us translates to char, but in the future might translate to wchar_t.